docs/source/API/io.rst

```diff
@@ -1,4 +1,4 @@
-.. automodule:: act.io
+.. automodule:: act.qc
```

Comment:
Ken, i think in the io.rst file you need to change this qc back to IO and then remove the "Clean" line.